src/nvim/getchar.c

```diff
@@ -181,18 +182,21 @@ static char_u *get_buffcont(buffheader_T *buffer,
   size_t count = 0;
   char_u          *p = NULL;
   char_u          *p2;
-  char_u          *str;
 
-  /* compute the total length of the string */
-  for (buffblock_T *bp = buffer->bh_first.b_next; bp != NULL; bp = bp->b_next)
+  // compute the total length of the string
+  for (const buffblock_T *bp = buffer->bh_first; bp != NULL; bp = bp->b_next) {
     count += STRLEN(bp->b_str);
+  }
```

Comment:
`const` isn't really needed for such small scopes. The compiler would figure it out, and it doesn't help a human reader because the scope is so small, so it is a net negative.